### DISCUSSION TITLE
Is it possible to add c# support or how to convert to C# dll

### DISCUSSION THREAD

**Discussion by squirrelfeng** [DIPlib/diplib#178] [General]
Hello, great lib, however is it possible  to provide some example to convert data into C# and work with opencvsharp？
Thanks!

**Comment by crisluengo** [comment]
I have never used C#, and haven’t used Windows in many years. So the best I can do is offer this link: https://stackoverflow.com/q/935664/7328782

I hope someone else comes by here that can offer more details.
